contracts/cw-core/src/contract.rs

```diff
@@ -562,8 +584,9 @@ pub fn query_dump_state(deps: Deps, env: Env) -> StdResult<Binary> {
     let config = CONFIG.load(deps.storage)?;
     let voting_module = VOTING_MODULE.load(deps.storage)?;
     let proposal_modules = PROPOSAL_MODULES
-        .keys(deps.storage, None, None, cosmwasm_std::Order::Descending)
-        .collect::<Result<Vec<Addr>, _>>()?;
+        .range(deps.storage, None, None, cosmwasm_std::Order::Descending)
+        .map::<StdResult<ProposalModule>, _>(|kv| Ok(kv?.1))
+        .collect::<Result<Vec<ProposalModule>, _>>()?;
```

Comment:
```suggestion
        .map(|kv| Ok(kv?.1))
        .collect::<StdResult<Vec<ProposalModule>>>()?;
```
@ezekiiel ez kill of turbofish